pkg/cmd/util/downloadrequest/downloadrequest.go

```diff
@@ -77,7 +85,9 @@ func Stream(ctx context.Context, kbClient kbclient.Client, namespace, name strin
 	}
 
 	wait.Until(checkFunc, 25*time.Millisecond, ctx.Done())
-
+	if downloadUrlTimeout {
+		return ErrDownloadRequestDownloadURLTimeout
+	}
 	if created.Status.DownloadURL == "" {
 		return ErrNotFound
 	}
```

Comment:
Timeout error now makes these lines dead code because the following (line 81) never executed.
```
if updated.Status.DownloadURL != "" {
			created = updated
			cancel()
		}
```

Reply:
Please advise if you would prefer separate error for timeout at all.
`return ErrNotFound` was unreachable prior to this PR anyways.

Reply:
Actually ErrNotFound was used many times after this.. so maybe just delete this
```suggestion
```

And ErrNotFound still used after these lines.

Reply:
That way we have explicit error for timeout getting DownloadURL populated vs http request file not found.